app/controllers/teams_controller.rb

```diff
@@ -27,6 +27,7 @@ def new
   def edit
     @team.sources.build(kind: 'blog') unless @team.sources.any?
     @conferences = conference_list
+    @conference = Conference.new
```

Comment:
I'd prefer to now assign another view variable, see my comment in the form partial 👇